lib/retirement_swap/storage.rb

```diff
@@ -0,0 +1,53 @@
+require 'sequel'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.